packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -277,8 +279,14 @@ class TransactionService {
            * Capture the TBT as span after observing for all long task entries
            * and once performance observer is disconnected
            */
-          if (tr.captureTimings && metrics.tbt.duration > 0) {
-            tr.spans.push(createTotalBlockingTimeSpan(metrics.tbt))
+          if (tr.captureTimings) {
+            if (metrics.tbt.duration > 0) {
+              tr.spans.push(createTotalBlockingTimeSpan(metrics.tbt))
+            }
+            /**
+             * TODO: define the proper place to store cls.
+             */
+            tr.addContext({ cls: metrics.cls })
```

Comment:
Now thats spec is kind of finalised, do you want to move it under experience and we can merge this PR?